src/bullsAndCows.js

```diff
@@ -27,6 +27,42 @@
  */
 function bullsAndCows(generatedNumber, enteredNumber) {
   // write code here
+  const storage = {};
+  const regex = /^\d{4}$/;
+  let cows = 0;
+  let bulls = 0;
+
```

Comment:
It's reasonable to create `yourAnswer` right here instead of variables `cows` and `bulls`